seed/go-sdk/seed.yml

```diff
@@ -2,6 +2,18 @@ irVersion: v40
 displayName: Go SDK
 image: fernapi/fern-go-sdk
 changelogLocation: ../../generators/go/sdk/versions.yml
+
+artifactMetadata:
+  preInstallScript:
+    - sudo yum -y install go
+    - mkdir /tmp/go
+    - echo "export GOPATH=\$HOME/go" >> ~/.bash_profile
+    - source ~/.bash_profile
```

Comment:
I don't think we'll need to explicitly configure the `GOPATH` here - we're just running the `go build` command to verify the program compiles.

I would try to simplify Go's `preInstallScript` to just:
```yaml
artifactMetadata:
  preInstallScript:
    - sudo yum -y install go
```

If for some reason we actually need to configure a `GOPATH`, I would just export the envrionment variable directly rather than writing to `~/.bash_profile` and `source`'ing from there:

```yaml
artifactMetadata:
  preInstallScript:
    - sudo yum -y install go
    - export GOPATH=\$HOME/go
```

Reply:
Updated!